spec.yaml

```diff
@@ -2,7 +2,7 @@ swagger: "2.0"
 info:
   title: Open Cloud Mesh API
   description: Open Cloud Mesh Open API Specification.
-  version: 1.1.0
+  version: 1.2.0
```

Comment:
For now I'd keep the version as is, we shall see when to tag a 1.2.0 version - which would include other features as well - in the future. Maybe the CS3 workshop can be a "good" moment.

Reply:
Fixed in the latest commit